### PR TITLE
Fix Multinode Neutron DNS domain

### PR DESCRIPTION
**Pull request by Alex-Welsh**

### CODE REVIEW

File: etc/kayobe/environments/ci-multinode/kolla/globals.yml [hunk 1]
```diff
@@ -42,6 +42,8 @@ nova_backend_ceph: "yes"
 neutron_bridge_name: "{{ vxlan_interfaces[0].device }}-ovs"
 neutron_external_interface: "{{ vxlan_interfaces[0].device }}"
 
+neutron_dns_domain: "{{ root_domain }}."
```
Comment (markgoddard) [inline etc/kayobe/environments/ci-multinode/kolla/globals.yml:45]: Something for the upgrade notes?

**Review by markgoddard — DISMISSED** [review]

File: etc/kayobe/environments/ci-multinode/kolla/globals.yml (hunk 1 above)
Comment (Alex-Welsh) [reply etc/kayobe/environments/ci-multinode/kolla/globals.yml:45]: reno or docs?

Comment (markgoddard) [reply etc/kayobe/environments/ci-multinode/kolla/globals.yml:45]: It looks like this new variable is required to keep an existing feature working? If so, upgrade docs.

Comment (Alex-Welsh) [reply etc/kayobe/environments/ci-multinode/kolla/globals.yml:45]: Added some docs

**Review by markgoddard — APPROVED** [review]

**Pull request merged**